deployments/edx/config/prod.yaml

```diff
@@ -1,15 +1,20 @@
+nfsPVC:
```

Comment:
this block (in both prod and staging) should look like this:

```
nfsPVC:
  nfs:
    export/edxhomes-2023-08-28/prod
```

s/prod/staging/ for `staging.yml`.